lib/storage/file.js

```diff
@@ -1019,12 +1019,18 @@ File.prototype.getSignedPolicy = function(options, callback) {
  *     link will expire.
  * @param {string=} options.extensionHeaders - If these headers are used, the
  *     server will check to make sure that the client provides matching values.
+ * @param {string=} options.responseDisposition - If you provide this value, 
```

Comment:
The link you included below (http://www.iana.org/assignments/cont-disp/cont-disp.xhtml) seems like it'd be really really useful to someone mucking with this particular header. Maybe we can reference that?